.github/workflows/build.yml

```diff
@@ -56,12 +52,12 @@ jobs:
             typ: alpine
             allow-failure: false
           -
-            image: alpine:edge
+            image: alpine:3.20
             typ: alpine
-            allow-failure: true
+            allow-failure: false
           -
-            image: debian:buster-backports
-            typ: debian
```

Comment:
removed buster-backports as buster is EOL for a long time now but still keeps logic in the code base.